### PR TITLE
Update Settings and Docker files to Molo 3 and Django 1.9

### PR DESCRIPTION
**Pull request by saeedmb8**

### CODE REVIEW

File: .deploy.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-buildscript: sideloader/build.sh
-postinstall: sideloader/postinstall.sh
+buildscript: gem/sideloader/build.sh
+postinstall: gem/sideloader/postinstall.sh
```
Comment (miltontony) [inline .deploy.yaml]: This should not change

File: docker/docker-start.sh [hunk 2]
```diff
@@ -5,7 +5,8 @@ set -e
 # ensure we create the locales first so that Django doesn't complain
 LANGUAGE_CODE=en ./manage.py compilemessages
 
-./manage.py migrate
+./manage.py migrate core 0031 --fake
```
Comment (miltontony) [inline docker/docker-start.sh]: Please add a note as to why this is done

File: docker/settings.py [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 import dj_database_url
 
-from .base import *
+from .production import *
```
Comment (miltontony) [inline docker/settings.py]: This should remain as `.base`

**Comment by miltontony** [comment]
Don't delete the TAG Manager settings value - use wagtail settings as the first prize and fallback to the settings value. We can remove it at a later stage - create an issue for the removal.

**Comment by saeedmb8** [comment]
@miltontony ready for review, I will Update the rest of the templates in a separate PR

File: gem/context_processors.py [hunk 4]
```diff
@@ -12,6 +13,8 @@ def default_forms(request):
 
 
 def add_tag_manager_account(request):
+    GTM = SiteSettings.for_site(request.site)
```
Comment (miltontony) [inline gem/context_processors.py]: this variable should be `site_settings` instead of `GTM`

File: gem/context_processors.py [hunk 5]
```diff
@@ -12,6 +13,8 @@ def default_forms(request):
...
     return {
-        'GOOGLE_TAG_MANAGER_ACCOUNT': settings.GOOGLE_TAG_MANAGER_ACCOUNT
+        'GOOGLE_TAG_MANAGER_ACCOUNT': (settings.GOOGLE_TAG_MANAGER_ACCOUNT or
```
Comment (miltontony) [inline gem/context_processors.py]: Please swap this 2 so that the wagtail value is  tried first

**Comment by miltontony** [comment]
👍

**Pull request merged**